lifecycle/src/main/scala/it/pagopa/interop/agreementprocess/lifecycle/AttributesRules.scala

```diff
@@ -40,7 +42,14 @@ object AttributesRules {
     consumerAttributes: Seq[VerifiedTenantAttribute]
   ): Boolean = attributesSatisfied(
     eServiceAttributes.verified,
-    consumerAttributes.filter(_.verifiedBy.exists(_.id == producerId)).map(_.id)
+    consumerAttributes
+      .filter(
+        _.verifiedBy.exists(v =>
+          v.id == producerId && ((v.renewal == REVOKE_ON_EXPIRATION && v.extensionDate
+            .exists(ed => ed.isAfter(OffsetDateTimeSupplier.get()))) || v.renewal == AUTOMATIC_RENEWAL)
+        )
+      )
+      .map(_.id)
```

Comment:
😵‍💫 I'm thinking whether it's possible to refactor it in a slightly more readable way

Reply:
I agree with @TonioGela, maybe moving 
`((v.renewal == REVOKE_ON_EXPIRATION && v.extensionDate.exists(ed => ed.isAfter(OffsetDateTimeSupplier.get()))) || v.renewal == AUTOMATIC_RENEWAL)`
in another function (eg: isExpired) could be helpful.

May I suggest to add a test for this part?